unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Tests/EditMode/SettingsPanelTests_EditMode.cs

```diff
@@ -0,0 +1,77 @@
+using DCL.SettingsPanelHUD;
+using DCL.SettingsPanelHUD.Sections;
+using DCL.SettingsPanelHUD.Widgets;
+using NSubstitute;
+using NUnit.Framework;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Tests
+{
+    public class SettingsPanelTests_EditMode
```

Comment:
We are not using the `Should` naming here. I think its better to do so to keep consistency with our other tests.

Reply:
Fixed in this new PR (in order to avoid mix things between the part I and II): https://github.com/decentraland/explorer/pull/1784